README.md

```diff
@@ -11,202 +11,273 @@ The goal of the project is to create a common HTML5 UI abstraction layer for the
+BSD-based systems:
 
-* [Rust](https://github.com/Boscop/webview-rs)
-* [Python](https://github.com/zserge/webview-python)
-* [Nim](https://github.com/oskca/webview)
-* [Haskell](https://github.com/lettier/webviewhs)
-* [C#](https://github.com/webview/webview_csharp)
-* [Janet](https://github.com/janet-lang/webview)
-* [Crystal](https://github.com/naqvis/webview)
-* [Java](https://github.com/shannah/webviewjar)
-* [Ruby](https://github.com/Maaarcocr/webview_ruby)
-* [Pascal](http://github.com/PierceNg/fpwebview)
+* FreeBSD packages: `pkg install webkit2-gtk3`
+* Execution on BSD-based systems may require adding the `wxallowed` option (see [mount(8)](https://man.openbsd.org/mount.8))  to your fstab to bypass [W^X](https://en.wikipedia.org/wiki/W%5EX "write xor execute") memory protection for your executable. Please see if it works without disabling this security feature first.
 
-Instructions for [Go](#webview-for-go-developers) and [C/C++](#webview-for-cc-developers) are included below.
+### Windows
 
-## Webview for Go developers
+Your compiler must support C++17 and we recommend to pair it with an up-to-date Windows 10 SDK.
 
-### Getting started
+For Visual C++ we recommend Visual Studio 2022 or later. We have a [separate section for MinGW-w64](#mingw-w64-requirements).
 
-Install this library with `go get`:
+Developers and end-users must have the [WebView2 runtime][ms-webview2-rt] installed on their system for any version of Windows before Windows 11.
+
+## Getting Started
+
+If you are a developer of this project then please go to the [development section](#development).
+
+Instructions here are written for GCC when compiling C/C++ code using Unix-style command lines, and assumes that you run multiple commands in the same shell. See the [MinGW-w64 requirements](#mingw-w64-requirements) when building on Windows.
+
+You will have a working app but you are encouraged to explore the [available examples][examples] and try the ones that go beyond the mere basics.
+
+Start with creating a new directory structure for your project:
 
 ```sh
-$ go get github.com/webview/webview
+mkdir my-project && cd my-project
+mkdir build libs "libs/webview"
+curl -sSLo "libs/webview/webview.h" "https://raw.githubusercontent.com/webview/webview/master/webview.h"
+curl -sSLo "libs/webview/webview.cc" "https://raw.githubusercontent.com/webview/webview/master/webview.cc"
+```
+
+### Windows Preperation
+
+The [WebView2 library][ms-webview2-lib] is required when compiling programs:
+
+```bat
+mkdir libs\webview2
+curl -sSL "https://www.nuget.org/api/v2/package/Microsoft.Web.WebView2" | tar -xf - -C libs\webview2
+copy /Y libs\webview2\build\native\x64\WebView2Loader.dll build
 ```
 
-Import the package and start using it:
+> **Note:** You need to distribute the `WebView2Loader.dll` file along with your app unless you link it statically but you must build our library and its examples using Visual C++ if so.
```

Comment:
This could be broken up into two sentences to make it more readable.

Suggested rephrasing:
> **Note:** You need to distribute the `WebView2Loader.dll` file along with your app unless you link it statically. If so, you must build our library and its examples using Visual C++.

Reply:
I've changed the wording.